### DISCUSSION TITLE
YouTube 双语字幕支持自定义样式

### DISCUSSION THREAD

**Discussion by shiquda** [immersive-translate/immersive-translate#1387] [Ideas]
### 新功能 | New Feature

RT，比如能够设置字体大小、颜色、间隔等

### 为什么您想要添加这些功能 | Why you want to add these features

目前的双语字幕中，不同语言的字体大小相同。如果能够让用户自定义设置不同的样式，比如英文字体大小较小，中文稍大（类似电影字幕那样），能够增强观看体验。

### 提议 | Proposal

加入一个类似“YouTube 双字幕”扩展程序的样式操控设置页面。
![Clip_2024-01-31_10-07-37](https://github.com/immersive-translate/immersive-translate/assets/104683621/abfd860e-4b31-4558-ae56-7409eb65437a)

**Comment by shiquda** [comment]
https://github.com/immersive-translate/immersive-translate/discussions/1287
原文和译文的位置如果也支持调换就更好了。

**Comment by theowenyoung** [comment]
很棒的idea，由于目前的实现是劫持原网站的字幕，所以只能在youtube自己的字幕设置里调整样式。

未来可能会换成自己实现的字幕条，这样就可以支持这些样式的设置了。
